packages/flutter/lib/src/widgets/spacer.dart

```diff
@@ -2,18 +2,23 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import '../../rendering.dart';
```

Comment:
need to be imported as `package:flutter/rendering.dart`

Reply:
Done.